objectFactory.go

```diff
@@ -44,6 +44,12 @@ func createObjectFactory(parent *channelOwner, objectType string, guid string, i
 		return nil
 	case "Electron":
 		return nil
+	case "FetchRequest":
+		c := &channelOwner{}
```

Comment:
maybe `return nil` for now?

Reply:
this is required, in a follow up this will be fixed